docs/sources/examples/IRIS_Example.md

```diff
@@ -0,0 +1,50 @@
+# IRIS Example
+
+The following code illustrates the usage of TPOT with the IRIS data set. 
+
+```python
+from tpot import TPOT
+from sklearn.datasets import load_iris
+from sklearn.cross_validation import train_test_split
+
+iris = load_iris()
+X_train, X_test, y_train, y_test = train_test_split(iris.data, iris.target, train_size=0.75)

```

Comment:
Please ensure that `test_size=0.25` is also in this call. See #52 for the reasoning.